edx/cc2olx
# [BD-27] [TNL-7456] Find OLX alternative and add support for Essay problem type

This PR adds support for converting the Common Cartridge Essay problem to OLX Open Assessment.

**JIRA tickets**: 
- https://tasks.opencraft.com/browse/BB-2949

~~**Discussions**:~~

**Dependencies**: None

~~**Screenshots**:~~

~~**Sandbox URL**:~~

**Merge deadline**: None

**Testing instructions**:

1. Convert a Common Cartridge file with Essay ``cc.essay.v0p1`` type problem
2. Import into an edX instance with **ORA** installed. 
3. Essay type problems should be shown as ORA problems.

**Author notes and concerns**:
1. There is no equivalent for ORA ``criterion`` in Common Cartridge. So default ``criterions`` are added.
2. There is no equivalent for Sample Solution of Common Cartridge in ORA. Thus a HTML block has been created from the sample solution and added before the problem.

**Reviewers**

~~**Settings**~~
